cmd/git-sync/main.go

```diff
@@ -503,3 +513,22 @@ func setupGitSSH(setupKnownHosts bool) error {
 
 	return nil
 }
+
+func setupGitCookieFile() error {
+	log.V(1).Infof("configuring git cookie file")
+
+	var pathToCookieFile = "/etc/git-secret/cookie_file"
```

Comment:
why is this path hard-coded?  Why not make this the value of the flag?

Reply:
I am following the pattern used for the ssh secrets. These are paths within the git-sync container, so the host path would be handled on the docker/k8s configuration end.

If we would like git-sync to provide a more flexible interface here (for non container usage, I would think, as I don't see the advantage of flexible paths within the container), that seems fine, but we should unify the treatment between the cookie-file and the ssh secrets, I would think?

Reply:
blech.  Volumes are weak in this area.  the dir /etc/git-secret can only be used by a single secret, so either this goes into docs alongside the SSH secret or you pick a different dir (if we want them to work together).  My guess is that they should not work together, so that restriction is OK?

Need to write a doc, please.